### PR TITLE
gui/lang: Fix position and update lang strings.

### PR DESCRIPTION
**Pull request by Croden1999**
update lang strings for the Thophy Collection, and fixed missing lang strings part of the "CalcTextSize" position for the User Management, fix my mistake in [#1635](https://github.com/Vita3K/Vita3K/pull/1635), plz Mr. @Zangetsu38  review this pr again, thx a lot.

# About
image
image
image
image

### CODE REVIEW

File: lang/de.xml [hunk 1]
```diff
@@ -247,7 +247,8 @@ Du kannst Trophäen in Anwendungen verdienen, die die Trophäen-Funktionen unter
 		<delete_user>Lösche Profil</delete_user>
 		<user_delete>Wähle das Profil, das gelöscht werden soll.</user_delete>
 		<user_delete_msg>Folgendes Profil wird gelöscht.</user_delete_msg>
-		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten und Trophäen gelöscht.</user_delete_message>
+		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten
```
Comment (Zangetsu38) [inline lang/de.xml:250]: revert this, using textwrap if you want using return line

File: lang/de.xml [hunk 2]
```diff
@@ -247,7 +247,8 @@ Du kannst Trophäen in Anwendungen verdienen, die die Trophäen-Funktionen unter
...
```
Comment (Zangetsu38) [inline lang/de.xml:250]: same here, no using return line, using all time textwrapped for this case

Comment (Croden1999) [reply lang/de.xml]: > same here, no using return line, using all time textwrapped for this case

finished.

File: lang/de.xml [hunk 3]
```diff
@@ -247,7 +247,7 @@ Du kannst Trophäen in Anwendungen verdienen, die die Trophäen-Funktionen unter
 		<delete_user>Lösche Profil</delete_user>
 		<user_delete>Wähle das Profil, das gelöscht werden soll.</user_delete>
 		<user_delete_msg>Folgendes Profil wird gelöscht.</user_delete_msg>
-		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten und Trophäen gelöscht.</user_delete_message>
+		<user_delete_message>Wenn du das Profil löscht, werden alle zugehörigen Nutzerdaten &#xA;und Trophäen gelöscht.</user_delete_message>
```
Comment (Zangetsu38) [inline lang/de.xml:250]: what is that ? juste revert change

File: lang/ja.xml [hunk 4]
```diff
@@ -262,8 +262,7 @@ Vita3Kをお楽しみください。</completed_setup>
 		<delete_user>ユーザーを消す</delete_user>
 		<user_delete>削除するユーザーを選択してください。</user_delete>
 		<user_delete_msg>下記のユーザーが削除されます。</user_delete_msg>
-		<user_delete_message>ユーザーが削除されると、そのユーザーのセーブデータ、
-トロフィーは削除されます。</user_delete_message>
+		<user_delete_message>ユーザーが削除されると、そのユーザーのセーブデータ、&#xA;トロフィーは削除されます。</user_delete_message>
```
Comment (Zangetsu38) [inline lang/ja.xml:265]: no using that

File: lang/ru.xml [hunk 5]
```diff
@@ -247,7 +247,7 @@
 		<delete_user>Удалить пользователя</delete_user>
 		<user_delete>Выберите пользователя, которого хотите удалить.</user_delete>
 		<user_delete_msg>Выбранный пользователь был удалён.</user_delete_msg>
-		<user_delete_message>Если Вы удалите пользователя, сохранённая информация, трофеи будут удалены.</user_delete_message>
+		<user_delete_message>Если Вы удалите пользователя, сохранённая информация, &#xA;трофеи будут удалены.</user_delete_message>
```
Comment (Zangetsu38) [inline lang/ru.xml:250]: ? i said using textwrapp no need add strange things

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**